e2-studio-star-rx72n-firmware/libs/rx_hcsr04/src/rx_hcsr04.c

```
@@ -1549,6 +1917,7 @@ rx_err_t rx_hcsr04_worker_deinit(void)
 rx_err_t rx_hcsr04_init(rx_hcsr04_t* handle, const rx_hcsr04_config_t* config)
 {
   rx_err_t err;
+  uint8_t  effective_priority = k_hcsr04_irq_priority_unset; /* Effective ICU priority (IRQ mode only) */
```

Clang (14.0.6)

[warning] variable 'effective_priority' is not initialized

(cppcoreguidelines-init-variables)